socorro/processor/processor_2015.py

```diff
@@ -40,26 +37,41 @@
 #               socorro.lib.transform_rules.Rule
 default_rule_set = [
     [   # rules to change the internals of the raw crash
-        "raw_transform",  # name of the rule
-        "processor.json_rewrite",  # a tag in a dotted-form
-        "socorro.lib.transform_rules.TransformRuleSystem",  # rule set class
-        "apply_all_rules",  # rule set class method to apply rules
-        ""  # comma delimited list of fully qualified rule class names
-    ],
-    [   # rules to transform a raw crash into a processed crash
-        "raw_to_processed_transform",
-        "processer.raw_to_processed",
-        "socorro.lib.transform_rules.TransformRuleSystem",
-        "apply_all_rules",
-        ""
-    ],
-    [   # post processing of the processed crash
-        "processed_transform",
-        "processer.processed",
-        "socorro.lib.transform_rules.TransformRuleSystem",
-        "apply_all_rules",
-        ""
-    ],
+        "transform_rules",
+        "socorro.processor.mozilla_transform_rules.ProductRewrite,"
```

Comment:
I'd be super if we could get rid of this "list of two strings, but the second string is really a comma-separated list of strings as a string" thing, too. Is it easy to convert it to a list of dicts, maybe? Something like this?:

```
default_rule_set = [
    {
        'name': 'transform_rules',
        'rules': [
            'socorro.processor.mozilla_transform_rules.RollIt',
            'socorro.processor.mozilla_transform_rules.PatIt',
            'socorro.processor.mozilla_transform_rules.MarkItWithABee',
            ...
        ]
    }
]
```